### DISCUSSION TITLE
I have set new nodes with netdata through docker compose, but no default alerts configurations

### DISCUSSION THREAD

**Discussion by Levisnkyyyy** [netdata/netdata#16049] [Q&A]
Hello,
I have an old node running with netdata, which has the default alert configurations somehow.
Now I have added two additional nodes through docker compose, but there's no default alert configurations.

I have looked into the files of the old node which has the alerts, to see if its configured there, specifically health.d folder which is empty, and netdata.conf which is also empty.
I have no idea how these default configs came up to the old node in the beginning, but I also want them for the new nodes.
What should I do?

**Comment by ilyam8** [comment]
Hey, all stock (default) configs are in `/usr/lib/netdata/conf.d/`. Alerts are in `/usr/lib/netdata/conf.d/health.d/`.

**Reply by Levisnkyyyy** [reply]
Thank you, I have found them, both in the old node and the new one. They do exist in files.
But, I still can't see them in "Alert Configuration" in the UI, I tried restarting the docker container, also removed and re-built it but it still doesn't show up

**Reply by MrZammler** [reply]
Hi @Levisnkyyyy 

This could be because of this -> https://github.com/netdata/netdata/pull/16050. Is it possible please to test with tommorow's nightly?

**Reply by Levisnkyyyy** [reply]
@MrZammler I will test it out and let you know.
